packages/design-system-twrnc-preset/src/Theme/Theme.providers.tsx

```diff
@@ -2,19 +2,17 @@ import React, { createContext, useState, useMemo } from 'react';
 import { useColorScheme } from 'react-native';
 import { create } from 'twrnc';
 
-import { ColorSet, ColorScheme, colorSetList } from '../twrnc-settings';
+import { ColorScheme, colorSetList } from '../twrnc-settings';
 
 import type { ThemeContextProps, ThemeProviderProps } from './Theme.types';
 import { Theme } from './Theme.types';
 import { generateTailwindConfig } from './Theme.utilities';
 
 export const defaultThemeContextValue: ThemeContextProps = {
-  tw: create(generateTailwindConfig(ColorSet.Brand, ColorScheme.Light)),
-  colorSet: ColorSet.Brand,
-  colorSetList: colorSetList[ColorSet.Brand][Theme.Light],
+  tw: create(generateTailwindConfig(ColorScheme.Light)),
+  colorSetList: colorSetList[ColorScheme.Light],
   theme: Theme.Light,
-  setColorSet: () => {},
-  setTheme: () => {},
+  setTheme: () => undefined,
```

Comment:
- Refactored to remove `colorSet` usage: `generateTailwindConfig` and context no longer depend on it.
- Removed `colorSet` from ThemeContext and provider.
- `colorSetList` now references the color scheme directly, omitting the `brand` key but returning the same object shape.

Reply:
I thought you also wanted to remove `colorSetList`?